root/etc/e-smith/events/actions/nethserver-tancredi-conf

```diff
@@ -11,3 +11,24 @@ GRANT SELECT on asterisk.ampusers TO 'tancredi'@'localhost' identified by '$db_p
 FLUSH privileges;
 EOF
 
+# set default.ini on first install
+if [[ ! -f /var/lib/tancredi/data/scopes/defaults.ini ]]; then
+    cp /usr/share/tancredi/data/scopes/defaults.ini /var/lib/tancredi/data/scopes/defaults.ini
```

Comment:
Use cp -v, or better `echo` a NOTICE that the action is initializing the defaults.ini file.